src/main/java/com/imsweb/validation/ValidatorContextFunctions.java

```diff
@@ -298,6 +298,42 @@ public Object fetchConfVariable(String id) throws ValidationException {
     }
 
     /**
+     * Logs the message
+     * <p/>
+     * Created on Dec 20, 2007 by depryf
+     * @param message Message to log
+     */
+    @ContextFunctionDocAnnotation(paramName1 = "message", param1 = "Message", desc = "Logs the given message.",
```

Comment:
All of the documentation is that way. param1 is the value that is shown in the help in DMS. I'm not sure what paramName is used for.